### PR TITLE
Create Database

### PR DESCRIPTION
**Pull request by lukman155**
* [x]  Create a database named vet_clinic.
* [x]  Create a table named animals with the following columns:
   id: integer
   name: string
   date_of_birth: date
   escape_attempts: integer
   neutered: boolean
   weight_kg: decimal
* [x]   Query to find all animals whose name ends in "mon".
1-mon-query

 * [x]    Query to List the name of all animals born between 2016 and 2019.
2-year

 * [x]  Query to list the name of all animals that are neutered and have less than 3 escape attempts.
3-less

 * [x]  Query to list the date of birth of all animals named either "Agumon" or "Pikachu".
4-year

 * [x]  Query to list name and escape attempts of animals that weigh more than 10.5kg
5-name

 * [x]  Query to find all animals that are neutered.
6-neuter

* [x]  Query to find all animals not named Gabumon.
7-not

* [x]  Query to find all animals with a weight between 10.4kg and 17.3kg
8-within

### CODE REVIEW

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @lukman155  :wave: ,
I am Nemwel your reviewer for this PR :man_beard: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## :heavy_multiplication_x:  Invalid Code Review Request :heavy_multiplication_x: 

You have submitted a project, that uses the wrong Gitflow. Your project currently merges changes into the main branch, while it is supposed to merge the changes from a feature branch into the dev branch. Kindly make this change.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,240 +1,60 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
-
```
Comment (HAMMAS-SALEEM) [inline README.md:6]: - [ ] You have submitted a project, that uses GitHub Flow but you are supposed to use Gitflow. To use Gitflow you follow these rules :arrow_down:
- You make a development branch from the main/master branch :+1:
- Then you make your development branch default :blush:
- At last, whenever you wanna make a change or project, or milestone then you make your feature branch from the development branch and after making that change you would merge that feature branch to development branch :+1:

File: README.md [hunk 2]
```diff
@@ -1,240 +1,60 @@
...
-<details>
-<summary>Database</summary>
   <ul>
     <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
   </ul>
 </details>
 
-<!-- Features -->
-
```
Comment (HAMMAS-SALEEM) [inline README.md:100]: - [ ] Your readme file is not descriptive enough :fearful:  Kindly make sure you have all these sections Title, description, Tech Stack, Key Features, Future Features, Getting Started Section, Authors Section, Contribution Section, Acknowledgment Section, Show your support Section, MIT link in your license file :+1:
- [ ] Make sure that all sections should have information and no llink is broken :+1:

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submission :red_circle:

## Hey @lukman155 :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following some requirements

Good job on implementing some of the requirements but I cannot proceed review of your pull request because there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**